packages/KInput/KInput.vue

```diff
@@ -9,6 +9,7 @@
       <div class="text-on-input">
         <label
           :for="inputId"
+          v-bind="labelAttributes"
```

Comment:
What's the purpose of binding the attributes here? Since it's a native HTML label, what would be the desired effect?

Reply:
You could also do something like apply a `data-testid` to the label.

Reply:
So if you add a `help` prop to a regular label, I'm assuming it doesn't error